models/semantic_models/ad_reporting__ad_report.yml

```diff
@@ -1,5 +1,5 @@
 semantic_models:
-  - name: ad_reporting__ad_report
+  - name: ad_report
```

Comment:
Curious why this is showing up since in `main` it is already "ad_report"? 🤔

Reply:
I think this is that issue Jamie mentioned with the rollbacks. It seems to be looking at the state of main when I created the branch. I think I'd have to rebase here to make it go away.

Reply:
This seems to have resolved itself when I resolved the merge conflicts.